components/eamxx/docs/technical/index.md

```diff
@@ -1,3 +1,23 @@
-# SCREAM Technical Guide
+# EAMxx Technical Guide
 
-SCREAM contributors and maintainers will add detailed technical information about SCREAM here.
+The goal of this document is to describe the specific equations, parameterizations, and numerical methods used in the current version of EAMxx. Because our master-branch implementation changes every time we make a new commit, this documentation will also evolve continuously. As such, documentation for master should always be considered to be preliminary and under construction. If you want trustworthy documentation, pull it from an official model release. 
```

Comment:
seems like "specific equations, parameterizations, and numerical methods" should be put into a list.

Also, maybe "relevant equations" is more accurate to what we hope to put here? I'm imaging there might be a page that uses a simple derivation to explain a fundamental concept of a particular scheme?

On that note - I'm curious how well we can write equations in markdown... how amazing would it be to write a block of latex!?

Reply:
regarding putting stuff in a list versus inline - I think this is a matter of preference and I'm not going to change it. 

regarding "relevant equations" - I wanted to emphasize that the tech doc describes the particular version of the code associated with the current git hash rather than talking about generic best practices for writing a process (which should be in the dev guide). So "specific" here refers to equations, params, and methods rather than just equations. Thus I don't want to change it. 

regarding latex equations - we can! See https://github.com/E3SM-Project/E3SM/blob/master/components/elm/docs/tech-guide/top_solar_parameterization.md as an example.